internal/services/healthcare/healthcare_dicom_data_source.go

```diff
@@ -86,6 +87,82 @@ func dataSourceHealthcareDicomService() *pluginsdk.Resource {
 				Computed: true,
 			},
 
+			"data_partitions_enabled": {
+				Type:     pluginsdk.TypeBool,
+				Computed: true,
+			},
+
+			"cors_configuration": {
```

Comment:
configuration is redundant,
```suggestion
			"cors": {
```

Reply:
Thanks, updated.